app/models/action_token.rb

```diff
@@ -7,7 +7,7 @@
 # A password reset key is valid for a limited time and may safely be left unused.
```

Comment:
Yea, leftovers from the original rename. These were initially accurate, but we should make it more general...